pkgs/development/python-modules/pytest-responses/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, python3
+, fetchFromGitHub
+,
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "pytest-responses";
+  version = "0.5.1";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "getsentry";
+    repo = "pytest-responses";
+    rev = "refs/tags/${version}";
+    hash = "sha256-6QAiNWCJbo4rmaByrc8VNw39/eF3uqFOss3GJuCvpZg=";
+  };
+
+  buildInputs = with python3.pkgs; [ pytest ];
+
+  propagatedBuildInputs = with python3.pkgs; [
+    responses
+    flake8
+  ];
+
+  doCheck = false;
+  checkPhase = ''
+    # allow to find the module helper during the test run
+    export PYTHONPATH=$PYTHONPATH:$PWD/tests
+    py.test tests
+  '';
```

Comment:
```suggestion
  buildInputs = [ pytest ];

  propagatedBuildInputs = [ responses ];

  nativeCheckInputs = [ pytestCheckHook ];
```

Reply:
Use `pytestCheckHook` instead of creating your own check phase unless you really need to. Removed `flake8`; it's not needed for building, runtime or testing.